Game/marvin.h

```diff
@@ -25,6 +25,8 @@ class Marvin {
       C_CamMode,
       C_ToogleCamDebug,
       C_ToogleCamera,
+      // Insert
```

Comment:
Nitpick: this comment doesn't give any new info - can be removed

Reply:
I thought you are using those comments to separate groups of commands, like camera specific and other commands.
If I just delete the line it will look like the insert command is part of the camera commands.
So I will just remove the comment and leave a empty line for visual distinction.